python_modules/libraries/dagster-elt/dagster_elt/asset_defs.py

```diff
@@ -0,0 +1,53 @@
+from typing import List, Iterable, Optional, Union
+from dagster import AssetsDefinition, asset, AssetKey, AssetExecutionContext
+from dagster._core.definitions.events import CoercibleToAssetKey
+from . import SlingMode, SlingResource, SlingSource, SlingTarget
+import re
+
+
+def build_sling_asset(
+    key: AssetKey,
+    sling_resource_key: str,
```

Comment:
Just assuming the resource key is "sling" would be consistent with other integrations. At the least, I'd make "sling" the default so that users don't need to specify it on their own.